specs/aab-store and build/aab.md

```diff
@@ -0,0 +1,59 @@
+# App Center Stores and Build support for Android App Bundles
+	
+This proposal will cover a new feature to to add support for Android App Bundles in App Center.
+This feature will only cover support for .aab packages in the Stores service, and Build.
+	
+## Problem Statement
+	
+Google has introduced AAB and it's been the recommended method to upload packages to the Play Store.
+
+Google has also announced that By August 1, 2019, all apps that use native code must provide a 64-bit version in addition to the 32-bit version in order to publish an update. 
+
+App Center today offers the ability to build and upload .apk files to Google Play Store. Users looking to start using bundles will not be able to upload them via App Center today.
+   
+## Goal
+Given that users are looking to utilize Android App bundles, they can use App Center to Build the bundle and distribute it to the Google Play store directly from App Center.
+	
+## Solution
+	
+### Build
+* Given an Android project where the configuration contains a bundle, then users will see a new setting under build configuration for building an Android Bundle package.
+
+* App Center will offer a new selector in the Android build configuration: "Also Build an Android Bundle".
+This will be pre-selected if the module selected in the configuration screen contains a "bundle {...}" block.
+
+* If a bundle will be built, the build will have the "android-app-bundle" tag after it is finished.
```

Comment:
This is implementation detail. Let's remove it

Reply:
Removed.